bitwarden_license/src/app/providers/manage/bulk/bulk-confirm.component.ts

```diff
@@ -12,7 +12,7 @@ import { BulkConfirmComponent as OrganizationBulkConfirmComponent } from 'src/ap
 import { BulkUserDetails } from 'src/app/organizations/manage/bulk/bulk-status.component';
 
 @Component({
-    templateUrl: '/src/app/organizations/manage/bulk/bulk-confirm.component.html',
+    templateUrl: '../../../../../../src/app/organizations/manage/bulk/bulk-confirm.component.html',
```

Comment:
Need to figure out if there's some way to get the absolute paths working again, this is a pain.

Reply:
Apparently the old absolute path was never explicitly supported, and was "accidentally fixed in 11.2". And there are no plans to bring back support. Keeping it as is for now. Might be a sign that we should either make the base component more generic, or duplicate the templates.